### ISSUE TITLE
Can I trainning on luggage？

### ISSUE THREAD

**Issue by yu9s** [fwtan/RerankingTransformer#20]
hello! Can I trainning on luggage？if I can do it,   how should I deal data?  I don't understand how your dataset is processed

**Comment by fwtan** [comment]
> hello! Can I trainning on luggage？if I can do it, how should I deal data? I don't understand how your dataset is processed

Can you clarify what dataset are you planning to use? Or are you aiming to create a new dataset?

**Comment by yu9s** [comment]
> > hello! Can I trainning on luggage？if I can do it, how should I deal data? I don't understand how your dataset is processed
> 
> Can you clarify what dataset are you planning to use? Or are you aiming to create a new dataset?

Yes, I trainning in a new dataset. The dataset includes various types of suitcases and bages. Now I have trained it. I want to input two pictures at the same time to determine wheith  they are the same suitcases.  In eval_rerank.py, I found that the input is a large dataset.  
Do it support inputting two images？ please
